### ISSUE TITLE
property-no-vendor-prefix

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#347]
add exception array, for unique value for some browsers, example 
```css
-webkit-touch-callout: none;
```

**Comment by jeddy3** [comment]
We've previously discussed adding exceptions to the prefix rules, and we originally felt that they were edge-case enough that they could just be handled with the _disabling comments_ e.g. `*/ stylelint-disable property-no-vendor */`. But as this is the second request, it seems like it'll be worthwhile adding them in and I think we'd totally accept a pull request with that change. @davidtheclark What do you think?

@evilebottnawi Fancy having a go at adding an `exceptions` array to `property-no-vendor`?

**Comment by davidtheclark** [comment]
I'd like to get to the bottom of this first ...

@evilebottnawi, if you add `-webkit-touch-callout` to your stylesheet is the rule complaining? Theoretically it *should not* complain, because that rule is [not on a standards track](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-touch-callout), so Autoprefixer should be ignoring it, so the rule should be ignoring it. So in my opinion, if it is complaining then that is a bug (hopefully one we can fix).

**Comment by davidtheclark** [comment]
@jeddy3 I'm still not sure that there's much use case for an exceptions if we have the rule working properly. Vendor-specific functions like `-webkit-touch-callout` should not warn. And it has got to be super rare that a person wants to, say, set `webkit-transform` to be one thing and `transform` to be another thing.

**Comment by jeddy3** [comment]
> Vendor-specific functions like -webkit-touch-callout should not warn

Oo, yes. I hadn't thought about that. Scratch what I originally said, I agree with you :)

**Comment by davidtheclark** [comment]
In 33a34332b8ce38a3efd15f91c19437bf5f3f0b60 I added a test and verified that `-webkit-touch-callout` will not in fact cause a warning.

I also added some more explanation to the documentation for the two `-no-vendor-prefix` rules, to hopefully clear up this misunderstanding.

**Comment by jeddy3** [comment]
Thanks!
